src/lib/y2storage/planned/devices_collection.rb

```diff
@@ -156,7 +163,14 @@ def stray_blk_devices
       #
       # @return [Array<Planned::Device>]
       def all
-        @all ||= partitions + disks + stray_blk_devices + vgs + lvs + mds + bcaches
+        @all ||= partitions +
+          disks +
+          stray_blk_devices +
+          vgs +
+          lvs +
+          mds +
+          bcaches +
+          nfs_filesystems
```

Comment:
NP: when I see this huge sum and also know that in past some customers have system with thousands of disks I start worrying about performance in that case. Maybe concat helps here to avoid recreation of resulting array multiple times?

Reply:
Ok, makes sense to me.